samples/highcharts/plotoptions/treemap-grouping-advanced/demo.js

```diff
@@ -0,0 +1,118 @@
+(async () => {
+
+    const dataJson = await fetch(
+        'https://cdn.jsdelivr.net/gh/highcharts/highcharts@8a1f8f0eb4/samples/data/africa-export-2021.json'
+    ).then(response => response.json());
+
+    const lightenDarkenColor = (col, amt) =>
```

Comment:
We have the [Highcharts.Color.brighten](https://api.highcharts.com/class-reference/Highcharts.Color#brighten) function that we use in some demos that looks a bit friendlier...